packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -47,6 +47,10 @@ class ConfigCommand extends FlutterCommand {
       help: 'Remove all configured features and restore them to the default values.',
       negatable: false,
     );
+    argParser.addFlag(
```

Comment:
Do we need a new flag since you've updated the usage footer? It seems like either doing `flutter config` or `flutter help config` would show the relevant information, and these are commands I would intuitively try.

Reply:
I think showing the full list directly would be good too. I've just started from keeping the old behavior first.